core/src/main/codegen/templates/Parser.jj

```diff
@@ -6176,6 +6176,21 @@ SqlNode BuiltinFunctionCall() :
         <RPAREN> {
             return SqlStdOperatorTable.EXTRACT.createCall(s.end(this), args);
         }
+    |
```

Comment:
Note that this will need to be ported to the parser inside of Bodo repository in order for this to have effect.

We should still make the change here, to keep them identical.